### DISCUSSION TITLE
Problems with manubot citing from orcid/ auto-cite.py

### DISCUSSION THREAD

**Discussion by Harrisonthow** [greenelab/lab-website-template#140] [Q&A] [answered]
Hello,

I'm trying to get auto-cite.py to generate a list of citations based off my PI's ORCID. However every time auto-cite.py runs, I am faced with this error:

  Source 47 of 47 - doi:10.1371/journal.pone.0063325

    Using Manubot to generate new citation

    [Errno 2] No such file or directory: 'manubot'

    Manubot could not generate citation

This happens for every source, it says there is no such file/directory as manubot. I have not found any help on the internet so I figured this would be a good place to ask

**Comment by vincerubinetti** [comment, marked as answer]
Is this the repo in question:
https://github.com/Harrisonthow/Beautiful-quorum

It looks like you are not using auto-cite in the standard way: [letting it run automatically with github actions via pull requests or committing to main](https://github.com/Harrisonthow/Beautiful-quorum/actions/workflows/auto-cite.yaml).

Thus I have to assume you're trying to run `auto-cite.py` locally on your computer, and you don't have the Manubot package.

Manubot isn't a standard package that comes with Python, so you have to install it via `requirements.txt`. Please see the (advanced) instructions for building citations locally:
https://github.com/greenelab/lab-website-template/wiki/Advanced-Editing#build-citations-locally

**Reply by Harrisonthow** [reply]
Hi Vincent, the repo in question is actually this one here (it was previously private) 

https://github.com/Harrisonthow/akkaya-lab

would there be any significant differences between my testing one (quorum) and the one that I’m working on?

**Reply by vincerubinetti** [reply]
I don't think there should be differences... though as a side note, GitHub Pages (what this template uses for hosting the website) charges a fee if you want to host from a private repository.

I am looking at your Akkaya lab repo, and I do not see [any of the auto-cite runs failing with the error you posted above](https://github.com/Harrisonthow/akkaya-lab/actions/workflows/auto-cite.yaml). Some of them are failing because your `.yaml` files are not in the appropriate format (which the error messages tell you), but I don't see any errors complaining that Manubot is not installed. Can you link to the specific place where you saw that error message?
